### ISSUE TITLE
Recaptcha v2 not being validated

### ISSUE THREAD

**Issue by danfickle** [strangerstudios/paid-memberships-pro#1490]
**Describe the bug**
Our site is under active credit card verification blitz because the recaptcha is not being validated (although it is shown on the checkout page). With some effort I was able to determine the following.

**To Reproduce**
1. Enable recaptcha v2 in settings.
2. Go to checkout page and do not tick captcha box.
3. No error is reported and the membership is created.

**Expected behavior**
Recpatcha to be validated upon submission of checkout form.

**Isolating the problem (mark completed items with an [x]):**
- [x] I have deactivated other plugins and confirmed this bug occurs when only Paid Memberships Pro plugin is active.
- [x] This bug happens with a default WordPress theme active, or [Memberlite](https://wordpress.org/themes/memberlite/).
- [x] I can reproduce this bug consistently using the steps above.

**WordPress Environment**
PHP 7.0 on Ubuntu 16.04, Wordpress 5.5.1, PMP 2.4.4, Twentysixteen theme.

**Debugging**
At https://github.com/strangerstudios/paid-memberships-pro/blob/4286d876b3aa5a24dbc314fbd0e84701b5c3852f/preheaders/checkout.php#L396

the condition is not evaluating as true because `$recaptcha` is empty.

The only way I could get it to work was add the following lines above:
````php
require_once(PMPRO_DIR . '/includes/lib/recaptchalib.php' );
$recaptcha = pmpro_getOption( 'recaptcha' );
$recaptcha_publickey = pmpro_getOption( 'recaptcha_publickey' );
$recaptcha_privatekey = pmpro_getOption( 'recaptcha_privatekey' );
````

Thanks in advance.

**Comment by danfickle** [comment]
I'm not sure whether issues are being monitored, we've discovered another related problem. Once a malicious actor has created an account they are free to verify as many credit cards as they like. This is because of the `if ( ! $skip_account_fields` condition and the captcha is not shown on the checkout page for members.

**Comment by bryansayler** [comment]
I've also had these credit card blitzes. Was any progress made on this?

**Comment by danfickle** [comment]
According to the changelog, both these issues are fixed as of version 2.5.5. Huge thanks! Closing now.
